lighthouse-cli/test/smokehouse/test-definitions/byte-efficiency/byte-config.js

```diff
@@ -34,6 +34,12 @@ const config = {
       'source-maps',
     ],
   }],
+  audits: [
+    {path: 'unused-javascript', options: {
```

Comment:
error happens locally for me, too, either running `yarn smoke byte` or a regular run with `--config-path=lighthouse-cli/test/smokehouse/test-definitions/byte-efficiency/byte-config.js`, which gets the error message back: ``Runtime error encountered: Unable to locate audit: `unused-javascript`.``

This needs to be

```suggestion
    {path: 'byte-efficiency/unused-javascript', options: {
```

Reply:
I  was running `yarn smoke perf` 🤦